src/controllers/dashboard/encodingsettings.js

```diff
@@ -218,6 +220,16 @@ $(document).on('pageinit', '#encodingSettingsPage', function () {
             page.querySelector('.fldIntelLp').classList.add('hide');
         }
 
+        if ((this.value == 'videotoolbox')) {
+            page.querySelector('.videoToolboxTonemappingOptions').classList.remove('hide');
+        } else {
+            page.querySelector('.videoToolboxTonemappingOptions').classList.add('hide');
+        }
+
+        if ((this.value == 'videotoolbox')) {
+            page.querySelector('.allowAv1EncodingOption').classList.add('hide');
```

Comment:
Can be merged into the above statement, also make sure to add the else clause too